generators/entity/index.js

```diff
@@ -803,6 +803,40 @@ class EntityGenerator extends BaseBlueprintGenerator {
         this.context.reactiveRegularEagerRelations = this.context.reactiveEagerRelations.filter(rel => rel.id !== true);
       },
 
+      /*
+       * Composed generators uses context ready for the templates.
+       */
+      composing() {
+        const context = this.context;
+        if (!context.skipServer) {
+          this.composeWithJHipster('entity-server', {
+            context,
+          });
+        }
+
+        if (!context.skipClient) {
+          this.composeWithJHipster('entity-client', {
+            context,
+            skipInstall: this.options.skipInstall,
+          });
+          if (this.jhipsterConfig.enableTranslation) {
+            this.composeWithJHipster('entity-i18n', {
+              context,
+              skipInstall: this.options.skipInstall,
+            });
+          }
+        }
+
+        if (
+          this.jhipsterConfig.gatlingTests &&
+          (this.jhipsterConfig.applicationType === 'gateway' || this.jhipsterConfig.applicationType === 'monolith')
+        ) {
+          this.composeWithJHipster('entity-gatling', {
+            context,
+          });
+        }
```

Comment:
Composing with entity-* is currently done at loading step.
But since this is a server only test, it should be composed by entity-server.